src/simularium/WebsocketClient.ts

```diff
@@ -100,6 +103,9 @@ export class WebsocketClient {
         this.logger = jsLogger.get("netconnection");
         this.logger.setLevel(jsLogger.DEBUG);
 
+        // Use wss when connected to simularium-engine, use ws otherwise
+        this.secureConnection = !(useOctopus || localBackendServer);
```

Comment:
suggestion: add "secureConnection" boolean to NetConnectionParams instead of passing useOctopus and localBackendServer in to this class.